.travis.yml

```diff
@@ -9,7 +9,7 @@ compiler:
 services:
 - docker
 
-arch:
+arch-travis:
```

Comment:
I like `archlinux` better, WDYT?

Reply:
Is a mater of taste - seems the same to me :smirk: